nautobot/extras/utils.py

```diff
@@ -267,6 +267,9 @@ def populate_model_features_registry():
         feature_name = lookup_conf["feature_name"]
         registry["model_features"][feature_name] = registry_items
 
+    if registry.get("populate_model_features_registry_called", False):
+        registry["populate_model_features_registry_called"] = True
```

Comment:
I don't think the `if` is really needed here:

```suggestion
    registry["populate_model_features_registry_called"] = True
```

Reply:
https://github.com/nautobot/nautobot/blob/9fe1ffdc6d23bbe86d8d2e9c3ae087233b15e8fc/nautobot/extras/registry.py#L16-L19

Without it I was getting bit by this.